lib/spree_signifyd.rb

```diff
@@ -20,7 +20,7 @@ def set_score(order:, score:)
 
   def approve(order:)
     order.contents.approve(name: self.name)
-    order.shipments.each { |shipment| shipment.ready! unless shipment.ready? }
+    order.shipments.each(&:ready!)
     order.updater.update_shipment_state
     order.save!
   end
```

Comment:
This whole method in my mind really isn't Signifyd-specific, and I wonder if there's a better place for it in Solidus somewhere.